src/components/common/ui/column-chart/ColumnChart.tsx

```diff
@@ -0,0 +1,124 @@
+import { FC } from 'react';
+import { Bar } from 'react-chartjs-2';
+import { Chart, ChartData, type ChartOptions, registerables } from 'chart.js';
+
+import styles from './ColumnChart.module.scss';
+
+Chart.register(...registerables);
+
+const response = {
+  name: 'Академічна доброчесність',
+  amount: 2,
+  type: 'AMOUNT',
+  mark: {
+    1: 10,
+    2: 23,
+    3: 6,
+    4: 2,
+    5: 5,
+    6: 13,
+    7: 21,
+    8: 14,
+    9: 2,
+    10: 16,
+  },
+};
+
+const ColumnChart: FC = () => {
+  const options: ChartOptions<'bar'> = {
+    maintainAspectRatio: false,
+    plugins: {
```

Comment:
The component doesn't accept any props, all of the values are hardcoded in the component itself. Instead you should create a test page and pass test data as props to component